docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1411,6 +1411,23 @@ How to **investigate** and **fix** it:
     - Check the number of in-memory series shown on the `Mimir / Tenants` dashboard for an approximation of the number of series that will be compacted once these blocks are shipped from ingesters.
     - Check the configured `compactor_split_and_merge_shards` for the tenant. A reasonable rule of thumb is 8-10 million series per compactor shard - if the number of series per shard is above this range, increase `compactor_split_and_merge_shards` for the affected tenant(s) accordingly.
 
+##### Persistent Volume Resizing Prerequisite - jsonnet
```

Comment:
I'm not sure about this formatting. Is this information an additional step in troubleshooting the store-gateway? If so, should it be integrated into the bullet points above?

Reply:
Updated [f8c8f15](https://github.com/grafana/mimir/pull/9939/commits/f8c8f158cf498de578713ea88dc6ddf709798781). I added it as a final bullet of the store-gateway `How to **investigate** and **fix** it:`. Let me know if it looks better now

Reply:
I think this looks great, @NickAnge !